rendermodules/utilities/pillow_utils.py

```diff
@@ -1,3 +1,13 @@
 from PIL import Image
 
 Image.MAX_IMAGE_PIXELS = None
+
+# imports needed for rough_align_qc
+import os
+import matplotlib
```

Comment:
Can this be in another module, e.g. `matplotlib_utils` so that things importing pillow's image don't also have to import matplotlib.

Reply:
created a new file

Reply:
This was later removed, so unresolving.  It looks like the build was failing because seaborn is imported before matplotlib and so the backend was not actually changed by `rendermodules.utilities.matplotlib_utils`
I think the only reason tests pass now is that the tests import from `rendermodules.utilities.pillow_utils` before seaborn is called and thus incidentally set the matplotlib backend.  Note, however, that this will not work in production as in e.g. a docker container the import statement shows:
```
# python -c "import rendermodules.em_montage_qc.rough_align_qc"
rendermodules/utilities/pillow_utils.py:9: UserWarning: 
This call to matplotlib.use() has no effect because the backend has already
been chosen; matplotlib.use() must be called *before* pylab, matplotlib.pyplot,
or matplotlib.backends is imported for the first time.

The backend was *originally* set to 'TkAgg' by the following code:
  File "<string>", line 1, in <module>
  File "rendermodules/em_montage_qc/rough_align_qc.py", line 13, in <module>
    import seaborn as sns
  File "/opt/conda/lib/python2.7/site-packages/seaborn/__init__.py", line 6, in <module>
    from .rcmod import *
  File "/opt/conda/lib/python2.7/site-packages/seaborn/rcmod.py", line 5, in <module>
    from . import palettes, _orig_rc_params
  File "/opt/conda/lib/python2.7/site-packages/seaborn/palettes.py", line 12, in <module>
    from .utils import desaturate, set_hls_values, get_color_cycle
  File "/opt/conda/lib/python2.7/site-packages/seaborn/utils.py", line 11, in <module>
    import matplotlib.pyplot as plt
  File "/opt/conda/lib/python2.7/site-packages/matplotlib/pyplot.py", line 71, in <module>
    from matplotlib.backends import pylab_setup
  File "/opt/conda/lib/python2.7/site-packages/matplotlib/backends/__init__.py", line 17, in <module>
    line for line in traceback.format_stack()


  mpl.use('Agg')

```

Reply:
changed the order... I guess the tests are passing now.